### PR TITLE
Admin LoginPage Added

### PR DESCRIPTION
**Pull request by jkyazhini**
-[x] Admin LoginPage Validation

### CODE REVIEW

File: src/main/java/in/yazhini/validator/AdminService.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package in.yazhini.validator;
+
+public class AdminService {
+
+	// validate the data in Adminuser
+      private AdminService () {
+	
+        }
+	public static boolean login(String userName, String password) {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/yazhini/validator/AdminService.java]: Add method comments

File: src/main/webapp/Login_action.jsp [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+if(valid){
+	out.println("Successfully LoggedIn");
+}
+else{
+	
+	String message="Invalid Login Credentials";
+	response.sendRedirect("Login.jsp?errorMessage=" + message);
+}
+%>
+
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/Login_action.jsp]: Replace "JSP" with "Servlet"

File: src/main/webapp/index.jsp [hunk 3]
```diff
@@ -1,19 +1,14 @@
-<%@page import="java.util.Date"%>
-<%@page import="java.time.LocalDate"%>
-<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
-<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt"%>
-<html>
+<!DOCTYPE html>
+<html xml:lang="java">
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/index.jsp]: Replace to <html lang="en"> ( dont use Java)

**Review by ghost — CHANGES_REQUESTED** [review]
Review comments added

File: src/main/java/in/yazhini/LoginActionSelvlet.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import in.yazhini.validator.AdminService;
+
+/**
+ * Servlet implementation class Login_actionSelvlet
+ */
+@WebServlet("/Login_actionSelvlet")
+public class LoginActionSelvlet extends HttpServlet {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/yazhini/LoginActionSelvlet.java]: LoginActionSelvlet - rename servlet.
Url - avoid underscore. (e.g: LoginServlet)

File: src/main/java/in/yazhini/LoginActionSelvlet.java [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+	private static final long serialVersionUID = 1L;
+
+	/**
+	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse
+	 *      response)
+	 */
+	protected void doGet(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {
+		// TODO Auto-generated method stub
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/yazhini/LoginActionSelvlet.java]: Removed unused code

File: src/main/java/in/yazhini/validator/AdminService.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
+package in.yazhini.validator;
+
+public class AdminService {
+
+
+		// validate the data in Adminuser
+
+	      private AdminService () {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/yazhini/validator/AdminService.java]: Format the code.

File: src/main/webapp/Login_action.jsp [hunk 7]
```diff
@@ -0,0 +1,33 @@
+<%@page import="in.yazhini.validator.AdminService"%>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/Login_action.jsp]: Delete the Login_action.jsp, since you have used LoginServlet.

File: src/main/webapp/index.jsp [hunk 8]
```diff
@@ -1,19 +1,14 @@
-<%@page import="java.util.Date"%>
-<%@page import="java.time.LocalDate"%>
-<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
-<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt"%>
-<html>
+<!DOCTYPE html>
+<html lang=en>
 <head>
 <title>MyApp</title>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/index.jsp]: Change title.

**Review by ghost — CHANGES_REQUESTED** [review]
Comments Added

File: src/main/webapp/Login.jsp [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+		
+		<form action="Login_action.jsp" method="post">
+		<br /> <label for="name">UserName: </label> <input type="text"
+			name="name" id="name" placeholder="userName" size="20" autofocus
+			required> <br>
+		<br> <label for="name">Password: </label> <input type="Password"
+			name="password" id="password" placeholder="Password" size="20" required>
+		<br>
+		<br> <input type="radio" name="role" id="A" required value="ADMIN">ADMIN
+		<br /> <input type="radio" name="role" id="C" required value="CUSTOMER">CUSTOMER<br><br/>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/Login.jsp:25]: <br/>

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**